include/touch_screen.h

```diff
@@ -39,8 +39,8 @@ BOOL TouchScreen_Touched(void);
  * @return TRUE if the touch screen is tapped, FALSE otherwise.
  */
 BOOL TouchScreen_Tapped(void);
-BOOL sub_020227A4(u32 * param0, u32 * param1);
-BOOL sub_020227C0(u32 * param0, u32 * param1);
+BOOL TouchScreen_HoldLocation(u32 *x, u32 *y);
+BOOL TouchScreen_TapLocation(u32 *x, u32 *y);
```

Comment:
```suggestion
BOOL TouchScreen_GetHoldState(u32 *x, u32 *y);
BOOL TouchScreen_GetTapState(u32 *x, u32 *y);
```

My logic here is that `Get[Hold/Tap]State` makes it clear that this does more than just return the coordinates of the hold/tap, which is true; it will also return `TRUE` or `FALSE` depending on if the screen is actually held/tapped.